src/Unleash/DefaultUnleash.cs

```diff
@@ -235,6 +243,52 @@ private IEnumerable<Constraint> ResolveConstraints(ActivationStrategy activation
             }
         }
 
+        public void ConfigureEvents(Action<EventCallbackConfig> callback)
+        {
+            if (callback == null)
+            {
+                Logger.Error($"UNLEASH: Unleash->ConfigureEvents parameter callback is null");
+                return;
+            }
+
+            try
+            {
+                var evtConfig = new EventCallbackConfig();
+                callback(evtConfig);
+                EventConfig = evtConfig;
+            }
+            catch (Exception ex)
```

Comment:
Is there something that can be raised here? This looks pretty harmless to me

Reply:
This particular one is probably very harmless, it's the callback where you configure all the handlers for the various events. But anything started in here that fails on a file lock or whatever will crash the startup of Unleash

Reply:
I can stand by that!